pkg/api/aim/runs.go

```diff
@@ -888,6 +890,31 @@ func SearchAlignedMetrics(c *fiber.Ctx) error {
 	return nil
 }
 
+// DeleteRun will update the run to the deleted lifecycle stage
+func DeleteRun(c *fiber.Ctx) error {
```

Comment:
Testme?

Reply:
there's a big refactor coming that will make this testable